src/main/java/net/neoforged/neoforge/registries/IRegistryExtension.java

```diff
@@ -65,14 +65,15 @@ default <C extends RegistryCallback<T>> void addCallback(Class<C> type, C callba
     }
 
     /**
-     * Adds an alias that maps from the name specified by <code>from</code> to the name specified by <code>to</code>.
+     * Adds an alias that maps from <code>alias</code> to <code>registeredName</code>.
      * <p>
-     * Any registry lookups that target the first name will resolve as the second name, if the first name is not present.
+     * Any registry lookups for <code>alias</code> will be redirected to <code>registeredName</code>,
+     * unless an object with <code>alias</code> is present in the registry.
```

Comment:
Okay, one issue though, this sentence doesn't make any sense without the names I used

"unless an object with alias is present in the registry." is not english. The parameter name has to contain `name` for it to make sense.

Reply:
This could easily be changed to: "unless an object with the name <code>alias</code> is present in the registry".

Reply:
@Shadows-of-Fire need your wisdom or blessings

Reply:
That proposed one is pretty verbose, and doesn't flow as well. I don't understand why we went with this `alias` / `registeredName` over `oldName` / `newName`. You gain some notion of perceived correctness, which comes back to bite you when you actually want to talk about it (like in this doc).

Reply:
I don't really have bandwidth to deal with it atm. I guess do whatever. The definition of why you want to not use old/new is effectively a misuse of this method (aliases are for conversion, not for convenience )